docs/integrations/imou.md

```
@@ -0,0 +1,123 @@
+
+import JsonLd from '@site/src/components/seo/JsonLd';
+
+Imou cameras (a brand of Dahua) are an affordable, widely used choice, and most of them work with Gladys Assistant.
+
+Imou cameras expose a standard **RTSP** stream, so they connect through the generic [camera integration](/docs/integrations/camera). Gladys talks to the camera directly on your local network, which means your video feed stays at home and never transits through the Imou cloud.
+
+## Imou RTSP URL
+
+Imou cameras expose two RTSP streams on port `554`: a high resolution **main** stream and a lighter **sub** stream. Replace `username`, `password` and the IP address with your own values:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)